openapi/src/test/groovy/io/micronaut/openapi/visitor/OpenApiParameterSchemaSpec.groovy

```diff
@@ -80,20 +81,16 @@ package test;
 
 import io.micronaut.http.HttpResponse;
 import io.micronaut.http.annotation.Controller;
-import io.micronaut.http.annotation.Get;
-import io.micronaut.http.annotation.Patch;import io.micronaut.http.annotation.Post;
-import io.micronaut.http.annotation.Put;
+import io.micronaut.http.annotation.Patch;
+import io.micronaut.http.annotation.QueryValue;
 import io.swagger.v3.oas.annotations.Parameter;
 import io.swagger.v3.oas.annotations.media.Schema;
 
-import java.time.Period;
-import java.util.Optional;
-
 @Controller("/path")
 class OpenApiController {
 
     @Patch
-    public HttpResponse<String> processSync4(@Parameter(schema = @Schema(type = "string", format = "uuid")) String param) {
+    public HttpResponse<String> processSync4(@Parameter(schema = @Schema(type = "string", format = "uuid")) @QueryValue String param) {
```

Comment:
Added lost annotation `@QueryValue`

Reply:
We support no-annotation parameters, which can also be query value, but for proper OpenApi it might be needed

Reply:
In this test before my hanges in field `in` was null value